routing/src/prefix.rs

```diff
@@ -722,4 +723,81 @@ mod tests {
 
         assert_eq!(u128::try_from(prefix_size1).unwrap(), 2u128.pow(8));
     }
+
+    #[test]
+    fn test_bolero_prefixsize_compare() {
```

Comment:
Let's discuss tomorrow, this may not be quite right because of overflow, see my comment about incomparable elements.  Otherwise everything looks good.

Reply:
I actually just made the changes and added a commit.  Let me know what you think.